### PR TITLE
[MIG][12.0] migrate product_expiry_simple

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: product_expiry_simple/models/stock_quant.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    expiry_date = fields.Date(related='lot_id.expiry_date', store=True)
+    expired = fields.Boolean(related='lot_id.expired')
+
+    # method copy/pasted from the official product_expiry module
+    # Copyright Odoo SA
+    @api.model
+    def _get_removal_strategy_order(self, removal_strategy):
+        if removal_strategy == 'fefo':
+            return 'expiry_date, in_date, id'
+        return super(StockQuant, self)._get_removal_strategy_order(
```
Comment (bealdav) [inline product_expiry_simple/models/stock_quant.py]:
```suggestion
        return super()._get_removal_strategy_order(
```

**Review by bealdav — CHANGES_REQUESTED** [review]
_

**Review by bealdav — COMMENTED** [review]
just some remarks pushed as last commit

**Comment by alexis-via** [comment]
Rebased. Should be ready for review/merge.

**Comment by alexis-via** [comment]
Travis is green, ready to merge

**Review by bealdav — APPROVED** [review]

**Comment by rousseldenis** [comment]
@alexis-via IMHO, translations should be excluded as :
- it should be done through weblate
- it concerns another module

**Comment by alexis-via** [comment]
@rousseldenis I don't understand your message about translation. I didn't do any translation in this module ; I only migrated it.

**Comment by rousseldenis** [comment]
> @rousseldenis I don't understand your message about translation. I didn't do any translation in this module ; I only migrated it.

This commit : 

https://github.com/OCA/stock-logistics-workflow/pull/532/commits/37fb919a09e815ac30bf9d376ee9ceee0fac8224

But looking closer, they were broken. So, ok

**Comment by rousseldenis** [comment]
> > @rousseldenis I don't understand your message about translation. I didn't do any translation in this module ; I only migrated it.
> 
> This commit :
> 
> [37fb919](https://github.com/OCA/stock-logistics-workflow/commit/37fb919a09e815ac30bf9d376ee9ceee0fac8224)
> 
> But looking closer, they were broken. So, ok

No, my preceding comment was ok. Remove translations for no_negative module from this.

File: stock_no_negative/i18n/fr.po [hunk 2]
```diff
@@ -23,15 +23,15 @@ msgstr ""
 #: code:addons/stock_no_negative/models/stock_quant.py:38
 #, python-format
 msgid " lot '%s'"
-msgstr "lot%s"
+msgstr "lot '%s'"
```
Comment (rousseldenis) [inline stock_no_negative/i18n/fr.po:26]: @alexis-via Could you remove changes in no_negative module from this PR ?

Comment (alexis-via) [reply stock_no_negative/i18n/fr.po:26]: Oups ! Well, this PR is closed now, so I don't think i'll fix it.

**Pull request closed without merging**